### DISCUSSION TITLE
When updating from 1.25.0 to 1.25.3, it is not displayed correctly

### DISCUSSION THREAD

**Discussion by cx20** [google/filament#5824] [Q&A] [answered]
I have updated the WebGL version of Filament from `v1.25.0` to `v1.25.3`.
As a result, the model is no longer displayed correctly.
Is it a matter of usage?
When I compared several versions locally, it seemed that the behavior changed between `1.25.0` and `1.25.1`.

[Filament v1.25.0 + DragonAttenuation.gltf Result:](https://rawcdn.githack.com/cx20/gltf-test/d58a83d519adce3a030571c4e018942110bc1190/examples/filament/index.html?category=tutorialModels&model=DragonAttenuation&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/181287177-5fc3131d-81c7-4d4a-8cab-53f8002050bd.png)

[Filament v1.25.3 + DragonAttenuation.gltf Result:](https://rawcdn.githack.com/cx20/gltf-test/3a5fcbd516889eb15dd8499214d2189819461aaa/examples/filament/index.html?category=tutorialModels&model=DragonAttenuation&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/181287237-ffce175d-e638-4d12-8bf8-2aa530584487.png)

[Filament v1.25.0 + Duck.gltf Result:](https://rawcdn.githack.com/cx20/gltf-test/d58a83d519adce3a030571c4e018942110bc1190/examples/filament/index.html?category=tutorialModels&model=Duck&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/181289190-e7726247-20ff-4fb2-8920-4f4e68344584.png)

[Filament v1.25.3 + Duck.gltf Result:](https://rawcdn.githack.com/cx20/gltf-test/3a5fcbd516889eb15dd8499214d2189819461aaa/examples/filament/index.html?category=sampleModels&model=Duck&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/181289311-1d24de28-93b5-4310-973f-3bcfd0e23d72.png)

**Comment by romainguy** [comment]
@prideout @bejado @pixelflinger

**Comment by prideout** [comment]
Thanks for the bug report, this is really interesting.  I cannot reproduce with our drag-and-drop WebGL viewer on "main" using `glTF-Sample-Models/2.0/Duck/glTF`.  I am using macOS, are you on Windows?  If possible, maybe you could try the Duck model on other platforms.

**Reply by cx20** [reply]
Hmm. That's really interesting.
Certainly, I usually use Windows.

I have tried on several PCs and in a Windows environment I seem to have the same problem with Chrome and Firefox.
Interestingly, I have tried Windows and macOS with Bootcamp on a MacBook Air and this problem seems to occur on Windows OS and not on macOS.

**Reply by cx20** [reply]
> our drag-and-drop WebGL viewer 

Is WebGL Viewer here?
When I tried it, it seemed that `v1.22.0` was being used.
https://google.github.io/filament/viewer/

**Reply by cx20** [reply]
I changed the ANGLE backend in Chrome to OpenGL as a test, and the problem was corrected.
![image](https://user-images.githubusercontent.com/1833979/181587981-036eca7f-9f75-494d-97aa-1c21fbea30f6.png)

Does this mean that the problem is related to Windows OS and DirectX?

**Reply by prideout** [reply]
This could be a bug in the GLES-to-DX11 translator in ANGLE.  @cx20 would you be willing to file a bug at https://bugs.chromium.org/?

**Reply by cx20** [reply]
I reported an issue below.
https://bugs.chromium.org/p/chromium/issues/detail?id=1348363

**Reply by romainguy** [reply]
Same issue reported here: https://github.com/google/filament/issues/5859

**Comment by cx20** [comment, marked as answer]
I tried again with Filament `v1.27.0`.
I have confirmed that this problem has been corrected in the latest version.

https://github.com/google/filament/releases/tag/v1.27.0

[Filament v1.27.0 + DragonAttenuation.gltf Result:](https://rawcdn.githack.com/cx20/gltf-test/b79aec14c61d1ce0f29eda32d437ea13512f6874/examples/filament/index.html?category=tutorialModels&model=DragonAttenuation&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/190515446-c3bb61d5-fde1-46c2-870f-ca93dac20578.png)

[Filament v1.27.0 + Duck.gltf Result:](https://rawcdn.githack.com/cx20/gltf-test/b79aec14c61d1ce0f29eda32d437ea13512f6874/examples/filament/index.html?category=sampleModels&model=Duck&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/190515530-a15f72d6-3cf9-4d0b-93e2-f2ce82ef4307.png)

**Reply by romainguy** [reply]
Thank you! And sorry for the regression.
